src/Tribe/Settings.php

```diff
@@ -630,8 +630,16 @@ public function generate_page(): void {
 				<?php
 				do_action( 'tribe_settings_above_form_element' );
 				do_action( 'tribe_settings_above_form_element_tab_' . $current_tab, $admin_page );
+				/**
+				 * Filter the classes for the settings form.
+				 *
+				 * @since TBD
+				 *
+				 * @param array<string> $form_class The classes for the settings form.
+				 */
+				$form_class = apply_filters( 'tribe_settings_form_class', [ 'tec-settings__' .$current_tab . '-tab--active' ], $admin_page );
```

Comment:
My usual preference 😅 

```suggestion
				$form_class = apply_filters( 'tribe_settings_form_class', [ "tec-settings__{$current_tab}-tab--active" ], $admin_page );
```